src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAppConfig.targets

```diff
@@ -0,0 +1,62 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.GenerateAppConfig.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="WriteAppConfig" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
+  <UsingTask TaskName="SetGeneratedAppConfigMetadata" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
+
+  <PropertyGroup>
+    <_GenerateAppConfigIntermediateAppConfig>$(IntermediateOutputPath)$(TargetFileName).withSupportedRuntime.config</_GenerateAppConfigIntermediateAppConfig>
+  </PropertyGroup>
+
+  <Target Name="GenerateAppConfig"
```

Comment:
I'm wondering if GenerateAppConfig is a good name. Probably better to say what in app config we're generating to match GenerateBindingRedirects. GenerateSupportedRuntime, perhaps? It is a common case for netfx apps to already have an app.config for other reasons so we aren't always generating one from scratch, but that's what the name reads like to me.

Reply:
Fixed, renamed target to GenerateSupportedRuntime and the task is called WriteAppConfigWithSupportedRuntime (since it will create one)